buildpacks/python/runtime/server.py

```diff
@@ -0,0 +1,24 @@
+import sys
+import os
+
+from flask import Flask, request
+from cloudevents.http import from_http
+
+serverDir = os.path.dirname(__file__)
+funcDir = os.path.realpath(os.path.join(serverDir, ".."))
+sys.path.append(funcDir)
+
+import func
+
+app = Flask(__name__)
+
+# create an endpoint at localhost:3000
+
+@app.route("/", methods=["POST"])
+def handler():
+  # create a CloudEvent
+  event = from_http(request.headers, request.get_data())
+  return func.main(event)
+
+if __name__ == "__main__":
+  app.run(port=8080)
```

Comment:
@lance Does this start debug only or production ready server?

Reply:
Currently just debug. A production server is still on the todo list.

Reply:
> Still needs a production runtime - at the moment, it's development mode only

Oh now I noticed.